### PR TITLE
Typo in java-docs

### PR DESCRIPTION
**Pull request by prasanth08**

### CODE REVIEW

File: processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java [hunk 1]
```diff
@@ -21,15 +21,15 @@
...
  * </p>
  * <p>
  * If no date format for mapping is specified predefined ISO* formatters from
- * {@link java.time.format.DateTimeFormatter} are used.
+ * {@link DateTimeFormatter} are used.
  * </p>
  * <p>
  * An overview of date and time types shipped with Java 8 can be found at
- * http://docs.oracle.com/javase/tutorial/datetime/iso/index.html.
+ * <a href="http://docs.oracle.com/javase/tutorial/datetime/iso/index.html">...</a>.
```
Comment (filiphr) [inline processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java:32]: I am not entirely sure that the 3 dots are enough. Perhaps we can some additional text here?

**Review by filiphr — COMMENTED** [review]
All changes look good @prasanth08. I have only one small comment about the link

File: processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java (hunk 1 above)
Comment (prasanth08) [reply processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java:32]: In my opinion the line before the link is self explanatory and we can just have link as it is and remove the 3 dots.

Comment (filiphr) [reply processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java:32]: You are right @prasanth08. Let's remove tat last sentence.

Comment (prasanth08) [reply processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java]: @filiphr Updated.

**Comment by prasanth08** [comment]
@filiphr It was an issue of the self closing tag . Fixed it 😀

**Comment by filiphr** [comment]
Thanks @prasanth08. I manually merged this in https://github.com/mapstruct/mapstruct/commit/68571de01bd9b06cef647a3d9ad073f0f0cd2456

**Pull request closed without merging**